### PR TITLE
Doc 1429 workload management

### PR DESCRIPTION
**Pull request by lennessyy**

### CODE REVIEW

File: modules/API/pages/built-in-endpoints.adoc [hunk 1]
```diff
@@ -3228,15 +3228,82 @@ curl --fail -u <my_username>:<my_password> -X POST "http://localhost:14240/gsqls
...
+      "startTime": "2022-08-10 21:07:28.638",
+      "expirationTime": "2022-08-10 21:07:44.638",
+      "url": "/query/Social/rngExample",
+      "elapsedTime": 3380
+    },
+    {
+      "requestid": "131087.RESTPP_1_1.1660165648723.N",
+      "startTime": "2022-08-10 21:07:28.723",
+      "expirationTime": "2022-08-10 21:07:44.723",
+      "url": "/query/NewSocial/rngExample",
```
Comment (venu-joshi) [inline modules/API/pages/built-in-endpoints.adoc:3283]: Can you change graph name and query names to be more appropriate and in sync with our existing documentation (new, brandnew sounds too cheesy).

File: modules/API/pages/built-in-endpoints.adoc [hunk 2]
```diff
@@ -3228,15 +3228,82 @@ curl --fail -u <my_username>:<my_password> -X POST "http://localhost:14240/gsqls
...
+<1> Each request to run a query sent to the REST server has a unique ID that can be used to look up the request in the system logs.
+<2> The time at which the query will time out.
+====
 
+==== Parameters
+No parameters.
+
+=== List running queries of a graph
+
+`GET /showprocesslist/{graph_name}`
```
Comment (venu-joshi) [inline modules/API/pages/built-in-endpoints.adoc:3299]: can you move this API ahead of showprocesslistall API (so they reflect the timeline on when they were implemented and as people prefer to use in the order of appearance, this /showprocesslist/<graph> is less intrusive than the new one).

**Comment by lennessyy** [comment]
Comments addressed

File: modules/API/pages/built-in-endpoints.adoc [hunk 3]
```diff
@@ -3279,10 +3280,80 @@ Response::
...
-==== Parameters:
+==== Parameters
 
 No Parameters.
 
+=== List all running queries
+`GET /showprocesslistall`
+
+This endpoint reports the statistics of all running queries across all graphs on a TigerGraph instance: the query's request ID, start time, expiration time, and the REST endpoint's URL.
+This includes installed queries. interpreted queries, as well as built-in queries.
```
Comment (venu-joshi) [inline modules/API/pages/built-in-endpoints.adoc:3291]: Line 3291 through 3294 are applicable for both the APIs (as they are identical).  How can we reflect that in the document (copy the same text to both API description)?

**Review by venu-joshi — APPROVED** [review]

File: modules/system-management/pages/manage-services.adoc [hunk 4]
```diff
@@ -13,7 +14,7 @@ If you are using Docker, for example, you must run `gadmin start all` after star
 
 Use `gadmin stop`, `gadmin start` or `gadmin restart` with individual services such as GSQL, Kafka, or RESTPP to control them separate from the others.
 
-The `restart` and `stop` commands prompt you for confirmation each time they are run. To avoid this, run the commands with the `-y` flag to bypass the confirmation.
+The `restart` and `stop` commands prompt you for confirmation each time they are run.To avoid this, run the commands with the `-y` flag to bypass the confirmation.
```
Comment (a-m-thomas) [inline modules/system-management/pages/manage-services.adoc:17]: Newline after period

File: modules/system-management/pages/management-commands.adoc [hunk 5]
```diff
@@ -103,9 +103,10 @@ dict          gsql          kafkaconn     ts3serv
 etcd          gui           kafkastrm-ll  zk
 ----
 
+[#_gadmin_config]
 === gadmin config
 
-`gadmin config` commands are used to manage the configuration for the TigerGraph system. To get a complete list of configuration parameters that are available, see xref:reference:configuration-parameters.adoc[Configuration Parameters].
+`gadmin config` commands are used to manage the configuration for the TigerGraph system.To get a complete list of configuration parameters that are available, see xref:reference:configuration-parameters.adoc[Configuration Parameters].
```
Comment (a-m-thomas) [inline modules/system-management/pages/management-commands.adoc:109]: Newline after period

File: modules/system-management/pages/workload-management.adoc [hunk 6]
```diff
@@ -0,0 +1,55 @@
+= Workload Management
+:description: Overview of workload management in TigerGraph.
+
+Certain TigerGraph operations, such as running online analytical processing (OLAP) queries that touch a lot of data, can be memory-intensive.
+TigerGraph provides the following mechanisms for you to manage workload in your TigerGraph instances.
+
+== Limit number of current built-in heavy queries
+TigerGraph has a few built-in queries that are memory-intensive.
```
Comment (a-m-thomas) [inline modules/system-management/pages/workload-management.adoc:8]:
```suggestion
TigerGraph has a few built-in queries that are memory-intensive, here referred to as "heavy."
```

**Review by alexthomasTG — APPROVED** [review]

**Pull request merged**